ViewModels/GameBoardViewModel.cs

```diff
@@ -21,10 +32,253 @@ internal class GameBoardViewModel : ViewModelBase
             }
         }
 
+
+        //property for building game board.
+        public CanvasControl GameBoardCanvas
+        { get; set; }
+
+
+
         public GameBoardViewModel( ) 
         {
             this._textStorlek = "25";
             Location nest = new Location(new List<Piece>(), Helpers.TypeOfLocation.Nest);
```

Comment:
same here, this is testing of the Location class...

Reply:
sure :)